build/travis/job2_AppImage/build.sh

```diff
@@ -29,17 +29,29 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build AppImage. Are we cross-compiling?
+# Build AppImage depending on arch specified in $1 if cross-compiling, else default build x86_64
 case "$1" in
-  --32bit )
+
+  --armhf )
+    shift
+    # build MuseScore inside debian x86-64 multiarch image containing arm cross toolchain and libraries
+    docker run -i -v "${PWD}:/MuseScore" ericfont/musescore:jessie-crosscompile-armhf /bin/bash -c "/MuseScore/build/Linux+BSD/portable/RecipeArm --build-only $makefile_overrides"
+    # then run inside fully emulated arm image for AppImage packing step (which has trouble inside multiarch image)
+    docker run -i --privileged multiarch/qemu-user-static:register
+    docker run -i -v "${PWD}:/MuseScore" --privileged ericfont/musescore:jessie-packaging-armhf /bin/bash -c "/MuseScore/build/Linux+BSD/portable/RecipeArm --package-only"
```

Comment:
Just a general formatting tip for scripts: try to limit the line length to 80 characters so it fits in a Terminal window. Use `\` if you have to go onto another line. 80 characters can be too restrictive sometimes (even with line breaks) so feel free to exceed it when you have to, but if you go much over 100 characters we'll start having to scroll sideways. (Some people even recommend 75 characters to make room for diff information and/or line numbers in the Terminal, but that can be quite hard to achieve! Having readable indentation is more important than the line limit in my opinion.)